bin/cli

```diff
@@ -26,6 +26,13 @@ const verboseLogger = !argv.options.verbose
   }
 
 const progressLogger = function () {
+  if (argv.options['created-from-version']) {
+    const version = splitSemvarVersion(argv.options['created-from-version'])
```

Comment:
is semvar a typo? semver? Maybe just splitVersion is fine

Reply:
Semvar is the convention we're breaking apart - if it's not semvar we shouldn't use that function so it's important it's in the name.

Reply:
Haha, you'r right. Semvar not semvar.

Reply:
I've fixed it and fixed an old example where I did the same thing.